test/server/cardImplementations/01_SOR/2-1BSurgicalDroid.spec.js

```diff
@@ -0,0 +1,54 @@
+const exp = require('constants');
+
+describe('2-1B Surgical Droid', function() {
+    integration(function() {
+        describe('2-1B Surgical Droid\'s healing', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: '21b-surgical-droid', exhausted: false },
```

Comment:
The default state is ready so the `exhausted: false` is unnecessary, unless you were doing it just to be explicit